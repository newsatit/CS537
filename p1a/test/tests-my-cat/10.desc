input same file multiple times
