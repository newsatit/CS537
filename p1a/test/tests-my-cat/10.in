your program should print this line 4 times
