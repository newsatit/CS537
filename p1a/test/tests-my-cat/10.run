./my-cat INPUT_DIR/10.in INPUT_DIR/10.in INPUT_DIR/10.in INPUT_DIR/10.in
