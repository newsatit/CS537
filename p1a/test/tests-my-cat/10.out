your program should print this line 4 times
your program should print this line 4 times
your program should print this line 4 times
your program should print this line 4 times
